feature/staticroute/staticroute_test.go

```diff
@@ -60,21 +61,25 @@ func TestAugment(t *testing.T) {
 			},
 		},
 	}, {
-		desc:   "Static route with one next-hop",
-		static: New().WithRoute("1.1.1.1", []string{"1.2.3.44"}),
-		inNI:   &oc.NetworkInstance{},
-		wantNI: &oc.NetworkInstance{
-			Protocol: map[oc.NetworkInstance_Protocol_Key]*oc.NetworkInstance_Protocol{
+		desc:   "Static route Multiple Next Hops",
```

Comment:
nit:
"Static route with multiple next-hops"

Reply:
I will get this in my next commit if thats ok. Don't want to lose the approval on a cosmetic change.